src/basic/unit-def.c

```diff
@@ -338,6 +338,7 @@ DEFINE_STRING_TABLE_LOOKUP(notify_access, NotifyAccess);
 
 static const char* const job_mode_table[_JOB_MODE_MAX] = {
         [JOB_FAIL]                 = "fail",
+        [JOB_MEEK]                 = "meek",
```

Comment:
maybe call this "additive"? or "augment"

Reply:
Not sure. I think "additive" kinda misses the part that this bails on conflict?

Reply:
"tentative" ?

Reply:
"lenient"?